demos/security-compare-asset-allocations/demo.js

```diff
@@ -0,0 +1,122 @@
+
+async function displaySecurityDetails (postmanJSON) {
+    const ids = ['F000015CH2', 'F0000143IP'],
+        idNames = {
+            'F000015CH2': 
+                'Schroder International Selection Fund Global Emerging Market Opportunities',
+            'F0000143IP': 'Fidelity Funds - Asia Pacific Opportunities Fund I-Acc-USD'
+        }
+
+
+    const connector = new HighchartsConnectors.Morningstar.SecurityCompareConnector({
+        postman: {
+            environmentJSON: postmanJSON
+        },
+        security: {
+            ids,
+            idType: 'MSID'
+        },
+        converter: {
+            type: 'AssetAllocations'
+        }
+    });
+
+    await connector.load();
+
+    const typeMapping = {
+        '1': 'Stocks',
+        '2': 'Bonds',
+        '3': 'Cash',
+        '4': 'Other Instruments',
+        '99': 'Unclassified'
+    };
+
+    const result = ids.map(id => 
+        connector.table.getRowObjects()
+            .map(item => ({
+                name: typeMapping[item['AssetAllocations_Type_' + id]],
+                y: item['AssetAllocations_MorningstarEUR3_N_' + id]
+            }))
```

Comment:
I think we can resign from this. Use `dataLabels.format` to apply the translation and columnAssignment to choose the right columns for charts